package.json

```diff
@@ -19,9 +19,10 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-icons": "^4.4.0",
-    "react-scripts": "^5.0.1",
+    "react-scripts": "4.0.3",
```

Comment:
Version down?

Reply:
There's a build error due to an unresolved path that is because of a change in the newer react scripts, so it was not able to find the crypto library in sjcl. I tried a bunch of solutions but this was the last and only one I could get to work.